test/testcafe/spec/ChallengeOktaVerifyPush_spec.js

```diff
@@ -57,9 +59,11 @@ test
     const pageTitle = challengeOktaVerifyPushPageObject.getFormTitle();
     const pushBtn = challengeOktaVerifyPushPageObject.getPushButton();
     const a11ySpan = challengeOktaVerifyPushPageObject.getA11ySpan();
+    const logoClass = challengeOktaVerifyPushPageObject.getBeaconClass();
     await t.expect(pushBtn.textContent).contains('Push notification sent');
     await t.expect(a11ySpan.textContent).contains('Push notification sent');
     await t.expect(pushBtn.hasClass('link-button-disabled')).ok();
+    await t.expect(logoClass).contains(BEACON_CLASS);
```

Comment:
I see in the challenge auto-push spec you compare to a string directly `custom-logo`, and here you compare to a constant. I think you should be consistent for both. Direct string is fine with me, but if you want a constant, let's name it something more specific to what it is `BEACON_CLASS` is very generic for the Beacon, it is not for challenge ov push

Reply:
i have use string directly now.